setup.py

```diff
@@ -35,6 +35,7 @@
             'fake_storage fake_driver = delfin.drivers.fake_storage:FakeStorageDriver',
             'dellemc unity = delfin.drivers.dell_emc.unity.unity:UNITYStorDriver',
             'dellemc vmax = delfin.drivers.dell_emc.vmax.vmax:VMAXStorageDriver',
+            'dellemc vnx = delfin.drivers.dell_emc.vnx_block.vnx_block:VnxBlockStorDriver',
```

Comment:
Why do we name it as vnx block ? same storage system can be used for file also! we can just name vnx so that NAS interfaces can be added later